test/integration/ThreeChainEndToEnd.t.sol

```diff
@@ -124,7 +130,7 @@ contract ThreeChainEndToEndDeployment is EndToEndFlows {
 
         vm.prank(INVESTOR_A);
         sB.spoke.crosschainTransferShares{value: GAS}(
-            sC.centrifugeId, POOL_A, SC_1, INVESTOR_A.toBytes32(), amount, SHARE_HOOK_GAS
+            sC.centrifugeId, POOL_A, SC_1, INVESTOR_A.toBytes32(), amount, 0, SHARE_HOOK_GAS
         );
         assertEq(shareTokenB.balanceOf(INVESTOR_A), 0, "Shares should be burned on chain B");
```

Comment:
I think we need to add here a check to confirm `onTransfer()` is called using the mock. Something like:

```
assertEq(snapshotHook.transfers(..) == amount)
```